src/components/DataFieldWithCopy.tsx

```diff
@@ -0,0 +1,77 @@
+import Clipboard from '@react-native-clipboard/clipboard'
```

Comment:
no interesting changes in this file, it was copied from ActionRequestPayload with the WC specific stuff removed